src/components/dashboard/modal/category.js

```diff
@@ -30,6 +35,17 @@ const NewCategoryModal = (props) => {
         }
     }
 
+    //press escape to exit the form, press enter to submit
+    function handler({ key }) {
+        if (ESCAPE_KEYS.includes(String(key))) {
+            props.toggleForm()
+        }
```

Comment:
In what cases will `key` be a number and what cases will it be the string?

Reply:
I downloaded firefox and did more testing on chrome, safari, and firefox. Looks like we only need to use the key names such as Enter, and don't need both Enter and 13 (keyboardEventValue). Just updated the changes :)